### PR TITLE
Remove instances of checkpointz

### LINKED ISSUE #531: Typo
On this page -> https://docs.teku.consensys.io/development/reference/cli#checkpoint-sync-url
This appears ->  URL of a Checkpointz endpoint

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Closes #531 

- I am not sure what the "checkpointz" mean, but the issue suggests it is a typo. There are some "checkpointz" in this doc: https://eth-clients.github.io/checkpoint-sync-endpoints/

PietjePuk | ✅ | ✅ | ❌ | https://checkpointz.pietjepuk.net
-- | -- | -- | -- | --

- This PR removes all mentions of "checkpointz" in the doc set.

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
@alexandratran @mjsmike62  PTAL

File: docs/get-started/checkpoint-start.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ When starting from a recent checkpoint, Teku downloads historic chain data in th
 You need access to a beacon node with [REST API enabled] (for example, Teku) to download the
 finalized checkpoint state file.
 
-Alternatively, you can use a Checkpointz endpoint from
+Alternatively, you can use a checkpoint endpoint from
```
Comment (joaniefromtheblock) [inline docs/get-started/checkpoint-start.md:20]:
```suggestion
Alternatively, you can use a checkpoint state endpoint from
```

File: docs/reference/cli/index.md [hunk 2]
```diff
@@ -232,7 +232,7 @@ checkpoint-sync-url: "https://beaconstate.ethstaker.cc"
   </TabItem>
 </Tabs>
 
-URL of a Checkpointz endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
+URL of a checkpoint endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```
Comment (joaniefromtheblock) [inline docs/reference/cli/index.md:235]:
```suggestion
URL of a checkpoint state endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```

File: versioned_docs/version-23.11.0/get-started/checkpoint-start.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ When starting from a recent checkpoint, Teku downloads historic chain data in th
 
 You need access to a beacon node with [REST API enabled] (for example, Teku) to download the finalized checkpoint state file.
 
-Alternatively, you can use a Checkpointz endpoint from [this community-maintained list of checkpoint state endpoints](https://eth-clients.github.io/checkpoint-sync-endpoints/).
+Alternatively, you can use a checkpoint endpoint from [this community-maintained list of checkpoint state endpoints](https://eth-clients.github.io/checkpoint-sync-endpoints/).
```
Comment (joaniefromtheblock) [inline versioned_docs/version-23.11.0/get-started/checkpoint-start.md:17]:
```suggestion
Alternatively, you can use a checkpoint state endpoint from [this community-maintained list of checkpoint state endpoints](https://eth-clients.github.io/checkpoint-sync-endpoints/).
```

File: versioned_docs/version-23.11.0/reference/cli/index.md [hunk 4]
```diff
@@ -887,7 +887,7 @@ checkpoint-sync-url: "https://beaconstate.ethstaker.cc"
   </TabItem>
 </Tabs>
 
-URL of a Checkpointz endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
+URL of a checkpoint endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```
Comment (joaniefromtheblock) [inline versioned_docs/version-23.11.0/reference/cli/index.md:890]:
```suggestion
URL of a checkpoint state endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```

File: versioned_docs/version-23.12.0/get-started/checkpoint-start.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ When starting from a recent checkpoint, Teku downloads historic chain data in th
 You need access to a beacon node with [REST API enabled] (for example, Teku) to download the
 finalized checkpoint state file.
 
-Alternatively, you can use a Checkpointz endpoint from
+Alternatively, you can use a checkpoint endpoint from
```
Comment (joaniefromtheblock) [inline versioned_docs/version-23.12.0/get-started/checkpoint-start.md:20]:
```suggestion
Alternatively, you can use a checkpoint state endpoint from
```

File: versioned_docs/version-23.12.0/reference/cli/index.md [hunk 6]
```diff
@@ -232,7 +232,7 @@ checkpoint-sync-url: "https://beaconstate.ethstaker.cc"
   </TabItem>
 </Tabs>
 
-URL of a Checkpointz endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
+URL of a checkpoint endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```
Comment (joaniefromtheblock) [inline versioned_docs/version-23.12.0/reference/cli/index.md:235]:
```suggestion
URL of a checkpoint state endpoint used to [start Teku from a recent state](../../get-started/checkpoint-start.md).
```

**Review by alexandratran — APPROVED** [review]
LGTM! I know you did a "replace all," but next time you only need to update docs in the `docs` folder (the development version of the docs).

**Comment by rolfyone** [comment]
it wasn't a typo... https://github.com/ethpandaops/checkpointz

**Pull request merged**